flink-libraries/flink-table/src/main/scala/org/apache/flink/table/functions/AggregateFunction.scala

```diff
@@ -35,6 +36,18 @@ abstract class AggregateFunction[T] extends UserDefinedFunction {
   def createAccumulator(): Accumulator
 
   /**
+    * Retract the input values from the accumulator instance.
+    *
+    * @param accumulator the accumulator which contains the current
+    *                    aggregated results
+    * @param input       the input value (usually obtained from a new arrived data)
```

Comment:
I think it should be ok to use "accumulator" here. Retract is a method, it retracts the value from the accumulator.